adapters_l1_attributes.go

```diff
@@ -0,0 +1,45 @@
+package monomer
+
+import (
+	"bytes"
+	"math/big"
+
+	"github.com/ethereum-optimism/optimism/op-node/rollup/derive"
+	ethtypes "github.com/ethereum/go-ethereum/core/types"
+)
+
+// global variable to avoid creating it every time
+var (
+	L1AttributesTx = ethtypes.DepositTx{
+		// From:                derive.L1InfoDepositerAddress,
+		To:                  &derive.L1BlockAddress,
+		Mint:                nil,
+		Value:               big.NewInt(0),
```

Comment:
Are these both bigints? Why nil for one and an instantiated 0 for the other?

Reply:
I decided to use [this approach](https://github.com/ethereum-optimism/optimism/blob/3542398896d9faca6b379fe67e3985d722cf80b6/op-node/rollup/derive/l1_block_info.go#L311)